docs/plugin-development.rst

```diff
@@ -389,6 +389,18 @@ You can use all of the testing utilities provided by the ``base.TestCase`` class
 from core. You will also get coverage results for your plugin aggregated with
 the main Girder coverage results if coverage is enabled.
 
+Plugins can also use the external data interface provided by Girder as described
+in :ref:`use_external_data`.  For plugins, the data key files should be placed
+inside a directory called ``plugin_tests/data/``.  When referencing the
+files, they must be prefixed by your plugin name as follows
+
+.. code-block:: cmake
+
+    add_python_test(my_test EXTERNAL_DATA plugins/cats/test_file.txt)
```

Comment:
Another reason, I'm building a new directory to store the key files is because *theoretically* the plugin directory doesn't have to be a subdirectory of girder.  In that case, inferring a relative path string (as required by `ExternalData`) to the plugin's data directory is difficult (or perhaps impossible).